README.md

```diff
@@ -43,6 +43,20 @@ Note that (for now), the compiled code ends up under `_build/default/lib/`,
 so you can symlink or copy `_build/default/lib/aeplugin_dev_mode` into the
 Aeternity plugin lib root (see below).
 
+## Loading
+
+After compiling, copy or link the application directory found at
+`_build/default/lib/aeplugin_dev_mode` into the AE node's plugin lib
+root. If not otherwise configured, this defaults to `$AETERNITY_TOP/plugins/`.
+After this, the node can be started either with a configuration as suggested
+below, or most simply by providing some OS environment variables, for example:
+
+```
+AE__SYSTEM__DEV_MODE=true \
+  AE__SYSTEM__PLUGINS='[{"name":"aeplugin_dev_mode"}]' \
```

Comment:
I was expecting to get rid from this as well, so perhaps I meant to implement the `system.dev_mode` option in the node itself ? 🤔 It starts to be a bit messy tho

Reply:
I was hoping that `system.dev_mode` would do that, as I stated maybe a misunderstanding.

`system.dev_mode` should not set the plugins defaults, but enable the plugin.
The plugin itself, should set all the defaults if enabled (that include beneficiary etc.), but that in case it's not overwritten by user configuration of course.